tests/gnrc_sixlowpan_frag_sfr_congure_impl/tests/01-run.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2021 Benjamin Valentin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import contextlib
+import subprocess
+import sys
+import time
+
+from subprocess import Popen
+from riotctrl_shell.gnrc import GNRCICMPv6Echo, GNRCICMPv6EchoParser, GNRCIPv6NIB
+from riotctrl_shell.netif import Ifconfig
+from riotctrl.ctrl import RIOTCtrlBoardFactory
+from riotctrl_ctrl import native
+from riotctrl_shell.netif import IfconfigListParser
+
+PARSERS = {
+    "ping6": GNRCICMPv6EchoParser(),
+    "ifconfig": IfconfigListParser(),
+}
+
+
+class RIOTCtrlAppFactory(RIOTCtrlBoardFactory):
+
+    def __init__(self):
+        super().__init__(board_cls={
+            'native': native.NativeRIOTCtrl,
+        })
+        self.ctrl_list = list()
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc):
+        for ctrl in self.ctrl_list:
+            ctrl.stop_term()
+
+    def get_shell(self, application_directory='.', env={'BOARD': 'native'}):
+        # retrieve a RIOTCtrl Object
+        ctrl = super().get_ctrl(
+            env=env,
+            application_directory=application_directory
+        )
+        # append ctrl to list
+        self.ctrl_list.append(ctrl)
+        # start terminal
+        ctrl.start_term()
+        ctrl.term.logfile = sys.stdout
+        # return ctrl with started terminal
+        return Shell(ctrl)
+
+    def get_shells(self, num=1):
+        terms = []
+        for i in range(num):
+            terms.append(self.get_shell())
+        return terms
+
+
+class Shell(Ifconfig, GNRCICMPv6Echo, GNRCIPv6NIB):
+    pass
+
+
+def first_netif_and_addr_by_scope(ifconfig_out, scope):
+    netifs = PARSERS["ifconfig"].parse(ifconfig_out)
+    key = next(iter(netifs))
+    netif = netifs[key]
+    return (
+        key,
+        [addr["addr"] for addr in netif["ipv6_addrs"] if addr["scope"] == scope][0],
+    )
+
+
+def global_addr(ifconfig_out):
+    return first_netif_and_addr_by_scope(ifconfig_out, "global")
+
+
+@contextlib.contextmanager
+def rpl_nodes(factory, zep_dispatch):
+    # linear topology with 4 nodes
+    topology = ("A B\n"
+                "B C\n"
+                "C D\n")
+    zep_dispatch.stdin.write(topology.encode())
+    zep_dispatch.stdin.close()
+
+    # create native instances
+    nodes = factory.get_shells(4)
+    A, _, _, D = nodes
+
+    netifs = PARSERS["ifconfig"].parse(A.ifconfig_list())
+    iface = next(iter(netifs))
+    # add prefix to root node
+    A.nib_prefix_add(iface, "2001:db8::/32")
+    A.ifconfig_add(iface, "2001:db8::1/32")
```

Comment:
```suggestion
    A.nib_prefix_add(iface, "2001:db8::/64")
    A.ifconfig_add(iface, "2001:db8::1/64")
```

should be sufficient here